cs/utils.texy

```diff
@@ -4,12 +4,22 @@ Utility
 .[perex]
 V balíčku `nette/utils` najdete sadu užitečných tříd pro každodenní použití:
 
-| [Rozšíření objektů v PHP | PHP Language Enhancements] | Nette\SmartObject
-| [Práce s obrázky | images] | Nette\Utils\Image
-| [Stránkování | Pagination] | Nette\Utils\Paginator
-| [Validace proměnných | validators] | Nette\Utils\Validators
-| [Práce s řetězci | strings] | Nette\Utils\Strings
-| [Práce s poli | arrays] | Nette\Utils\Arrays
-| [Datum a čas |Date and Time] | Nette\Utils\DateTime
-| [Generování HTML kódu | HTML elements] | Nette\Utils\Html
-| [Parsování a generování JSON | json] | Nette\Utils\Json
+Instalace
+=========
+
+/--
+composer require nette/utils
+\--
+
+Užitečné třídy
+==============
+
+| [Rozšíření objektů v PHP | PHP Language Enhancements] | [Nette\SmartObject |api:]
```

Comment:
I think it makes the table unclear, ie. it is not entirely clear which link to actually click on.